src/UnitTest/tests/Mesh/TestDerivedData.h

```diff
@@ -2418,6 +2418,707 @@ void DerivedData_object::test< 57 >() {
+                                      17, 29,  2, 10,         
+                                      30, 27, 22, 31,         
+                                      21, 28, 25, 31,         
+                                      28, 30, 25, 31,         
+                                      27, 17, 14, 26,         
+                                       4, 21, 25, 11,         
+                                      29, 27, 31, 26,         
+                                      30, 19, 25, 15,         
+                                      29, 21, 10, 31,         
+                                      17, 29, 10, 26,         
+                                      22, 27, 14, 31,         
+                                      30, 22, 15, 31,         
+                                      27, 14, 31, 26,         
+                                      21, 25, 11, 31,         
+                                      25, 30, 15, 31,         
+                                      21, 10, 31, 11,         
+                                      10, 29, 31, 26,         
+                                      22, 15, 31,  7,         
+                                      14, 22, 31,  7,         
+                                      15, 25, 31, 18,         
+                                      25, 11, 31, 18,         
+                                      10, 31, 11,  3,         
+                                      31, 10, 26, 18,         
+                                      14, 31, 26,  7,         
+                                      15, 31,  7, 18,         
+                                      11, 31, 18,  3,         
+                                      31, 10, 18,  3,         
+                                      31, 26,  7, 18  };
+
+  // boundary elements for this mesh
+  std::map< int, std::vector< std::size_t > > belem {
+    { { 0 }, {  1, 
+                1,
+                2,
+                2,
+                3,
+                4,
+                5,
+                5,
+                6,
+                6,
+               12,
+               14,
+               16,
+               17,
+               18,
+               19,
+               25,
+               28,
+               29,
+               29,
+               30,
+               31,
+               32,
+               33,
+               34,
+               35,
+               36,
+               43,
+               44,
+               45,
+               46,
+               50,
+               51,
+               51,
+               53,
+               55,
+               61,
+               63,
+               64,
+               65,
+               66,
+               67,
+               68,
+               69,
+               70,
+               71,
+               72,
+               73  } },
+  };
+
+  // Shift elem IDs to start from zero
+  for (auto& iss : belem)
+  {
+    for (auto& f : iss.second)
+    {
+      f = f-1;
+    }
+  }
```

Comment:
I think this could work using `tk::shiftToZero()`. E.g.,
```
for (auto& iss : belem) tk::shiftToZero( iss.second );
```

Reply:
Yes it did work. Changed it.